mappings/net/minecraft/item/ItemStackHolder.mapping

```diff
@@ -0,0 +1,24 @@
+CLASS net/minecraft/class_5630 net/minecraft/item/ItemStackHolder
```

Comment:
Is this implemented by a lot of classes like `Inventory`, or is this an interface with very few selected implementations? If all implementations are mainly focused on stacks, the `getStack` and `setStack` can just be `get` `put`/`set`

This name... sounds a bit weird, honestly. I'd really name it `ItemSlot` or so.

Reply:
I had a big write-up here but decided that mapping this class is simply outside the scope of this PR. I'll remove it.